.github/workflows/build-docker-image-and-binaries.yaml

```diff
@@ -36,59 +37,102 @@ jobs:
           submodules: recursive
 
       - name: Install Cosign
+        if: matrix.os == 'ubuntu-18.04' && matrix.arch == 'amd64'
```

Comment:
Why not sign all binaries?

Reply:
because it is related to docker images. Binaries are signed in gpg section